release-notes/8.0/preview/api-diff/rc1/Microsoft.NETCore.App/8.0-rc1_System.Runtime.Intrinsics.Wasm.md

```diff
@@ -0,0 +1,12 @@
+# System.Runtime.Intrinsics.Wasm
```

Comment:
CC. @radekdoulik 

Were these APIs intentionally removed?

Reply:
@radekdoulik @lewing can you take a look?